src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -224,52 +224,53 @@ public static bool NotUsingHttps(string url)
         [RavenAction("/admin/databases", "PUT", AuthorizationStatus.Operator)]
         public async Task Put()
         {
-            var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name").Trim();
-            if (ResourceNameValidator.IsValidResourceName(name, ServerStore.Configuration.Core.DataDirectory.FullPath, out string errorMessage) == false)
-                throw new BadRequestException(errorMessage);
-
             var raftRequestId = GetRaftRequestIdFromQuery();
-
-            if (LoggingSource.AuditLog.IsInfoEnabled)
-            {
-                var clientCert = GetCurrentCertificate();
-
-                var auditLog = LoggingSource.AuditLog.GetLogger("DbMgmt", "Audit");
-                auditLog.Info($"Database {name} PUT by {clientCert?.Subject} ({clientCert?.Thumbprint})");
-            }
-
+            
             ServerStore.EnsureNotPassive();
             using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
             using (context.OpenReadTransaction())
             {
                 var index = GetLongFromHeaders("ETag");
                 var replicationFactor = GetIntValueQueryString("replicationFactor", required: false) ?? 1;
-                var json = context.ReadForDisk(RequestBodyStream(), name);
+                var json = context.ReadForDisk(RequestBodyStream(), "Database Record");
                 var databaseRecord = JsonDeserializationCluster.DatabaseRecord(json);
+                
                 if (databaseRecord.Encrypted)
                     ServerStore.LicenseManager.AssertCanCreateEncryptedDatabase();
-
-                if (string.IsNullOrWhiteSpace(databaseRecord.DatabaseName))
-                    throw new ArgumentException("DatabaseName property has invalid value (null, empty or whitespace only)");
-                databaseRecord.DatabaseName = databaseRecord.DatabaseName.Trim();
-
-                if (databaseRecord.Settings.TryGetValue(RavenConfiguration.GetKey(x => x.Core.DataDirectory), out var dir))
+                
+                // Validate Directory
+                var dataDirectoryThatWillBeUsed = databaseRecord.Settings.TryGetValue(RavenConfiguration.GetKey(x => x.Core.DataDirectory), out var dir) == false ?
+                                         ServerStore.Configuration.Core.DataDirectory.FullPath : PathUtil.ToFullPath(dir, ServerStore.Configuration.Core.DataDirectory.FullPath);
```

Comment:
Let's use instead of PathUtil a `PathSetting`

Reply:
done